templates/shared/verifyicon.tmpl

```diff
@@ -0,0 +1,115 @@
+{{/*
+Template Attributes:
+* Commit: Commit object
+* Verification: Verification struct
+*/}}
+{{if (and .Commit .Commit.Signature .Verification)}}
+	{{$icon := ""}}
+	{{$text := ""}}
+	{{$colorClass := ""}}
+	{{if .Verification.Verified}}
+		{{$icon = "octicon-verified"}}
+		{{if eq .Verification.TrustStatus "trusted"}}
+			{{$colorClass = "tw-text-green"}}
+			{{$text = (ctx.Locale.Tr "sig_verified")}}
+		{{else if eq .Verification.TrustStatus "untrusted"}}
+			{{$colorClass = "tw-text-orange"}}
+			{{$text = (ctx.Locale.Tr "repo.commits.signed_by_untrusted_user")}}
+		{{else}}
+			{{$colorClass = "tw-text-orange"}}
+			{{$text = (ctx.Locale.Tr "repo.commits.signed_by_untrusted_user_unmatched")}}
+		{{end}}
+	{{else}}
+		{{$icon = "octicon-unverified"}}
+		{{if .Verification.Warning}}
+			{{$colorClass = "tw-text-red"}}
+		{{else}}
+			{{$colorClass = "tw-text-inherit"}}
+		{{end}}
+		{{if .Verification.Reason}}
+			{{$text = (ctx.Locale.Tr .Verification.Reason)}}
+		{{else}}
+			{{$text = (ctx.Locale.Tr "sig_failed")}}
```

Comment:
Not sure if this condition can be reached. At least it wasn't necessary before, so maybe can remove.